playbooks/templates/Tranco_top_domains_to_observables.json

```
@@ -0,0 +1,170 @@
+{
+  "name": "Tranco top domains to observables",
+  "nodes": {
+    "0": {
+      "name": "Fetch Tranco (weekly)",
```

The frequency will depend of the setting the user chooses in the trigger configuration

```suggestion
      "name": "Fetch Tranco",
```